doc/_utils/redirects.py

```diff
@@ -21,41 +21,62 @@
     ("app_boards", "config_and_build/board_support"),
     ("app_dev/board_support/index", "config_and_build/board_support"),
     ("config_and_build/board_support", "config_and_build/board_support/index"),
+    ("config_and_build/board_support/index", "app_dev/board_support/index"),
```

Comment:
These are covered further down in the file, so I'm not sure what is wrong.

Reply:
I can see that pages like https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16636/nrf/config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib.html are not redirecting to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16636/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib.html. This is because the following entries are missing: 

(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_quick_start, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_quick_start)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_adding, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_partitioning, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_partitioning )
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_dfu_image_versions, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_dfu_image_versions)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_config, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_config)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_signature_keys, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_signature_keys)
(config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_downgrade_protection, app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_downgrade_protection)